spec/cocoapods-integration-specs

```diff
@@ -1 +1 @@
-Subproject commit 3c74dd99cf532ba61336e940f79fb8b02467a65b
```

Comment:
but what changed in the integration specs, since there are as of yet no functional changes in the CocoaPods codebase?